scripts/run-test.sh

```diff
@@ -39,6 +39,7 @@ echo "Running test on ${PROJECT_PATH}"
 cd $PROJECT_PATH
 go mod init test || true
 go install . || true
+go test -timeout=30m
```

Comment:
Can you find a way to link this value to the [job timeout](https://github.com/nearform/k8s-kurated-addons/blob/main/.github/workflows/preview.yaml#L20) in GitHub? At the moment this test will be killed after 15m. Thank you!

Reply:
Will align timeouts manually for now.